src/Traits/ReflectionFunctionLikeTrait.php

```diff
@@ -198,6 +198,10 @@ public function getReturnType(): ?\ReflectionType
             return null;
         }
 
+        if ($returnType === 'iterable') {
+            $returnType = 'Traversable|array';
+        }
```

Comment:
@lisachenko this is needed as php 8.2 display `Traversable|array` for `iterable, see https://3v4l.org/V1RCP